Sources/FormattingHelpers.swift

```diff
@@ -1651,6 +1651,24 @@ extension Formatter {
 
 /// Helpers for recursively traversing the declaration hierarchy
 extension Formatter {
+    /// Applies `forEachRecursiveDeclarations` in place
```

Comment:
A description like this seems better:

```suggestion
    /// Recursively calls the `operation` for every declaration in the source file
```